scaleway/resource_lb.go

```diff
@@ -71,6 +71,7 @@ func resourceScalewayLb() *schema.Resource {
 				Optional:    true,
 				Default:     false,
 				Description: "Release the IPs related to this load-balancer",
+				Deprecated:  "The resource ip will be destroyed by it's own resource. Please set this to `false`",
```

Comment:
Could you include a migration guide in case this resource is deprecated? For instance by showing an example with a scaleway_lb that got it to true and show how to migrate with the now two distinct resources.

Reply:
Thank you,  I've just added.